cocos/core/algorithm/binary-search.ts

```diff
@@ -28,7 +28,7 @@
  * Searches the **ascending sorted** array for an element and returns the index of that element.
  * @param array The array to search in.
  * @param value The value to search.
- * @return The index of the searched element in the sorted array, if found;
+ * @returns The index of the searched element in the sorted array, if found;
```

Comment:
@return is the correct syntax, please don't modify the syntax, otherwise it won't be recognized

Reply:
VSCode uses `returns` if use `/**` to generate comments. So i modified existing usage too.